### ISSUE TITLE
Suggestion: throw error when resize option is not present

### ISSUE THREAD

**Issue by ChaseGiunta** [vvo/gifify#60]
As it seems that the `--resize` option is a requirement (seen in my experience and #23) , maybe throw an error if the parameter is not present in command. Unless this is a bug? Gifify will continue to run indefinitely, and not create a write file in location, if not the `--resize` option is not defined

**Comment by vvo** [comment]
Hi @ChaseGiunta --resize is not always needed strangely it seems. Only in some cases, would you mind digging a bit more to see why?

**Comment by ChaseGiunta** [comment]
Unfortunately I'm not skilled enough to do so, ha. I can provide the video files (.mp4) if that is of some use.

**Comment by vvo** [comment]
Yes drop them here so that someone can have a look. I have really no idea on what's going on

**Comment by ChaseGiunta** [comment]
Apologies, it looks like it **_does_** create a temp file... But it will continue to run indefinitely (I think, I didn't leave it go to because it quickly ate up 5GB for this 7mb .mp4) https://cl.ly/0F3C2S3G1a1I

**Comment by vvo** [comment]
Maybe there's an underlying bug somewhere in the dependencies stack
